### DISCUSSION TITLE
How can hold State in React?

### DISCUSSION THREAD

**Discussion by mauryajayk** [community/community#140477] [Projects and Issues]
How can hold State in React?

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @mauryajayk ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by monwarqx1250** [comment]
there are hooks avaialble for managing state in react. you can also manage in class component by the state property. you can also use libraries like redux for managing global state
